src/components/common/YourCompany/EntityName.vue

```diff
@@ -16,7 +16,10 @@
         <label :class="{'error-text': invalidSection}">
           <strong>{{ getResource.entityReference }} Name</strong>
         </label>
-        <v-col md="1">
+        <v-col
+          md="1"
+          class="pa-0"
```

Comment:
This looked fine for SR - I did a Change of Name, double checking the other entity types now